### PR TITLE
feat(discord-utilities): add more limits

### PR DESCRIPTION
**Pull request by noftaly**
- Add limits related to upload size
- Add limit about maximum number of reactions
- Add limits about moderation
- Add limit about the biography length

Who made the biography length **190** characters ??? just why ??

### CODE REVIEW

**Review by favna — DISMISSED** [review]

File: packages/discord-utilities/src/lib/limits.ts [hunk 1]
```diff
@@ -137,7 +140,40 @@ export const MessageLimits = {
 	/**
 	 * Maximum characters allowed in a single message for a nitro user.
 	 */
-	MaximumNitroLength: 4000
+	MaximumNitroLength: 4000,
+
+	/**
+	 * Maximum numbers of reactions allowed for a message.
+	 */
+	MaximumReactions: 20,
```
Comment (c43721) [inline packages/discord-utilities/src/lib/limits.ts:148]: Out of all numbers, you chose 20?

**Review by c43721 — CHANGES_REQUESTED** [review]

File: packages/discord-utilities/src/lib/limits.ts (hunk 1 above)
Comment (favna) [reply packages/discord-utilities/src/lib/limits.ts:148]: These are discord API limits. Not our own limits.

Comment (noftaly) [reply packages/discord-utilities/src/lib/limits.ts:148]: You cannot add more than 20 different reactions to a message

**Review by c43721 — DISMISSED** [review]

File: packages/discord-utilities/src/lib/limits.ts (hunk 1 above)
Comment (c43721) [reply packages/discord-utilities/src/lib/limits.ts:148]: I was thinking something else. Ignore this :)

File: packages/discord-utilities/src/lib/limits.ts [hunk 2]
```diff
@@ -137,7 +140,40 @@ export const MessageLimits = {
...
+	 * Maximum upload size for a nitro user, in any guild or in DMs.
+	 * Size is in bytes, and correspond to 10MB.
+	 */
+	MaximumNitroUploadSize: 100_000_000,
+
+	/**
+	 * Maximum upload size for a free user for all different boost levels available in a guild.
+	 * Sizes are in bytes, and correspond to 8MB, 8MB, 50MB, and 100MB.
+	 */
+	MaximumUploadSizeInGuild: [8_000_000, 8_000_000, 50_000_000, 100_000_000]
```
Comment (vladfrangu) [inline packages/discord-utilities/src/lib/limits.ts:166]: Clever

File: packages/discord-utilities/src/lib/limits.ts [hunk 3]
```diff
@@ -137,7 +140,40 @@ export const MessageLimits = {
...
+} as const;
+
+/**
+ * Namespace containing limits related to built-in moderation features.
+ */
+export const ModerationLimits = {
+	/**
+	 * Maximum duration of a guild Timeout, in seconds (corresponds to 28 days).
```
Comment (vladfrangu) [inline packages/discord-utilities/src/lib/limits.ts:174]:
```suggestion
	 * Maximum duration of a guild timeout, in seconds (corresponds to 28 days).
```

File: packages/discord-utilities/src/lib/limits.ts [hunk 4]
```diff
@@ -157,5 +193,10 @@ export const UserLimits = {
 	/**
 	 * Maximum numbers of users in a DM group.
 	 */
-	MaximumUsersPerDMGroup: 10
+	MaximumUsersPerDMGroup: 10,
+
+	/**
+	 * Maximum characters allowed in a user's biography (the "About Me" section).
+	 */
+	MaximumBiographyLength: 190
```
Comment (vladfrangu) [inline packages/discord-utilities/src/lib/limits.ts:201]: We can keep it but this isn't something that we can alter via api calls as bot users 😛

**Review by vladfrangu — CHANGES_REQUESTED** [review]

File: packages/discord-utilities/src/lib/limits.ts (hunk 4 above)
Comment (favna) [reply packages/discord-utilities/src/lib/limits.ts:201]: Yet. Pretty sure Ian said in December's stage event that bots may get biographies in the future.

File: packages/discord-utilities/src/lib/limits.ts (hunk 2 above)
Comment (noftaly) [reply packages/discord-utilities/src/lib/limits.ts:166]: Useful if you want to get the one corresponding to your guild, you can use the tier as an index ^^

**Review by vladfrangu — APPROVED** [review]

**Review by favna — APPROVED** [review]

**Pull request merged**